storage/campaignfailedlog.go

```diff
@@ -0,0 +1,46 @@
+package storage
+
+import (
+	"fmt"
+
+	"github.com/segmentio/ksuid"
+
+	"github.com/mailbadger/app/entities"
+)
+
+// CreateReport creates a report.
+func (db *store) CreateCampaignFailedLog(l *entities.CampaignFailedLog) error {
```

Comment:
Do we actually need this method? It's a bit confusing to have CreateCampaignFailedLog and LogFailedCampaign methods since we'll most likely only be using one of them.

Reply:
@FilipNikolovski Yes you are right.